src/Sarif.Viewer.VisualStudio/SarifViewerPackage.cs

```diff
@@ -31,6 +31,7 @@ namespace Microsoft.Sarif.Viewer
     [ProvideService(typeof(ISarifLocationTaggerService))]
     [ProvideService(typeof(ITextViewCaretListenerService<>))]
     [ProvideService(typeof(ISarifErrorListEventSelectionService))]
+    [ProvideService(typeof(IErrorListTableControlProvider))]
```

Comment:
should we change to `IErrorListTableControlService`?

Reply:
I removed it at David's suggestion. If I had kept it, I would have preferred the name "Provider" since that's what it does -- provide the control. It's a common naming pattern in VS.

---
In reply to: [538264056](https://github.com/microsoft/sarif-visualstudio-extension/pull/313#discussion_r538264056) [](ancestors = 538264056)